classes/controller/FrontController.php

```diff
@@ -1984,9 +1963,47 @@ protected function getAlternativeLangsUrl()
         }
 
         foreach ($languages as $lang) {
-            $alternativeLangs[$lang['language_code']] = $this->context->link->getLanguageLink($lang['id_lang']);
+            $langUrl = $this->context->link->getLanguageLink($lang['id_lang']);
+            $alternativeLangs[$lang['language_code']] = $this->sanitizeUrl($langUrl);
         }
 
         return $alternativeLangs;
     }
+
+    /**
+     * Sanitize / Clean params of an URL
+     *
+     * @param string $url URL to clean
+     * @return string cleaned URL
+     */
+    private function sanitizeUrl($url) 
```

Comment:
```suggestion
    private function sanitizeUrl(string $url): string
```
Because it's develop :heart: